Form-Controls/index.html

```diff
@@ -7,21 +7,73 @@
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <section>
+      <label for="name">Name</label>
+      <input id="name" type="text" minlength="3" maxlength="8" required />
```

Comment:
Does this meet the requirements as listed in the readme? 

What is the benefit of limiting the user's name length? What happens if someone has a name longer than 8 characters?

https://shinesolutions.com/2018/01/08/falsehoods-programmers-believe-about-names-with-examples/